support/ci/fast_pass.sh

```diff
@@ -37,16 +36,19 @@ else
   echo
   echo "Checking for changed files since last merge commit:"
   echo
+  # shellcheck disable=2086
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2086

These have to be disabled since we need to treat these variables like arrays and pass them to `echo_indented` as separate arguments. Since they're passed in as environment variables we can't use proper bash arrays.